SCClassLibrary/Common/Collections/linux/extLinuxPlatform_findTerminalEmulator.sc

```diff
@@ -0,0 +1,38 @@
++ LinuxPlatform {
+
+    *findTerminalEmulator {|action|
+        var supportedTerminalEmulatorsCmds = [
+            // format args are title, shell and command (.shellQuoted)
+            "konsole --hold --title '%' -e % -c %",
+            "xterm -hold -T '%' -e % -c %",
+            "urxvt --hold -T '%' -e % -c %",
+            "rxvt --hold -T '%' -e % -c %",
+            "termite --hold --title '%' -e \"% -c %\"",
+            // Note: gnome, xfce4, mate and terminator can't be kept open via switch
+            // users could append ';read' to their commands
+            "gnome-terminal -t '%' -- % -c %",
```

Comment:
Isn't gnome-terminal's switch for command `-e` instead of `-c`? Just reading the man page, haven't tested

Reply:
From gnome-terminal's output
# Option “-e” is deprecated and might be removed in a later version of gnome-terminal.
# Use “-- ” to terminate the options and put the command line to execute after it.

Reply:
Oh go it, sorry for the noise!